### DISCUSSION TITLE
Slides do not load

### DISCUSSION THREAD

**Discussion by ilazaric** [community/community#110785] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Not sure if this is specific to me, but on:
https://github.com/CppCon/CppCon2016/blob/master/Tutorials/Template%20Normal%20Programming%2C%20Part%202/Template%20Normal%20Programming%2C%20Part%202%20-%20Arthur%20O'Dwyer%20-%20CppCon%202016.pdf
as I keep scrolling and loading more slides, at slide 25 it no longer has a button at the bottom center of the page, but a loading animation, that never finishes

**Comment by dakwahcrypto** [comment]
reload tabs not ended sirr , i don't acsess to live or space . back to sign in again and again
![IMG_1676](https://github.com/community/community/assets/160866963/58cb16fa-45b9-4ec1-8c9e-ec95abb359cd)
